remoteworkitem/trackeritem_repository_test.go

```diff
@@ -102,7 +103,8 @@ func TestConvertExistingWorkItem(t *testing.T) {
 
 		return err
 	})
-
+	db.Unscoped().Delete(&tq)
```

Comment:
This can be removed. This is what the gormsupport.DeleteCreatedEntities(db) does.